frontend/src/components/TodoTaskList/types.ts

```diff
@@ -0,0 +1,11 @@
+export type TodoTaskProps = {
+  text: string;
+  onStarClick(): void;
```

Comment:
co oznacza nazwa tej funkcji?

Reply:
Ogólnie to zerknąłem na branch, gdzie jest robiony task Todotask i skopiowałem typ, który stworzył Patryk i stworzyłem na podstawie tego typy do mojego taska. Nie wiem co ta funkcja będzie robiła. Jak Patryk skończy task to się dowiem.

Reply:
nazwa nie jest zbyt generyczna, dodatkowo polecam komunikować się z innymi osobami z zespołu żeby takie kwestie dogadywać